src/ActorsPanel.tsx

```diff
@@ -112,6 +112,9 @@ export const ActorsPanel: React.FC = () => {
                   ? 'line-through'
                   : undefined
               }
+              title={`Actor: ${
```

Comment:
I see what you are doing here since Cypress won't find text spanning multiple elements by default. However, this IMHO is functionally not needed and in fact, might be a small annoying distraction to the user. It only contains a pretty much duplicated text that is already visually available. 

Rule of thumb -> don't add code to only facilitate tests (which sometimes can be broken, since it's just sometimes impossible)